javascript-carousel/main.js

```diff
@@ -0,0 +1,73 @@
+var $image = document.querySelector('img');
+var $leftBumper = document.querySelector('.fa-chevron-left');
+var $rightBumper = document.querySelector('.fa-chevron-right');
+var $ballColumn = document.querySelector('.cycle-column');
+var $cycleCircleList = document.querySelectorAll('.ball');
+var $pressedBall = null;
+var i = 0;
+var imageSrcArr = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var lastSrcListIndex = imageSrcArr.length - 1;
+
+var intervalID = setInterval(cyclePokemon, 3000);
+
+function cyclePokemon() {
+  i++;
+  if (i === 5) {
+    i = 0;
+  }
```

Comment:
Instead of using a magic hard-coded number here, you should make use of the `length` of the array that contains the values you want to loop over.